core/interfaces.go

```diff
@@ -155,6 +155,8 @@ type BBs interface {
 	BBsCreateMessages(inputMessages []model.InputMessage) ([]model.Message, error)
 	BBsDeleteMessages(l *logs.Log, serviceAccountID string, messagesIDs []string) error
 	BBsSendMail(toEmail string, subject string, body string) error
+	BBsAddRecipients(l *logs.Log, messageID string, orgID string, appID string, userID string, mute *bool, read *bool) ([]model.MessageRecipient, error)
+	BBsDeleteRecipients(l *logs.Log, id string, appID string, orgID string) error
```

Comment:
You should remove many recipients to a message but not single one. You have to also pass the message id which you will delete recipeints for.